src/15_classes.py

```diff
@@ -2,20 +2,55 @@
 # constructor
 
 # YOUR CODE HERE
+class LatLon:
+    "This class takes a 'lat' and a 'lon"
+
+    def __init__(self, lat="Some latitude", lon="Some longitude"):
+        self.lat = lat
+        self.lon = lon
 
 # Make a class Waypoint that can be passed parameters `name`, `lat`, and `lon` to the
 # constructor. It should inherit from LatLon. Look up the `super` method.
 
 # YOUR CODE HERE
 
+
+class Waypoint(LatLon):
+    "This class inherits from LatLon and takes an additional 'name' parameter"
+
+    def __init__(self, name="Some place"):
+        self.name = name
+        super().__init__(self)
```

Comment:
It's a good practice to call `super().__init__` first thing inside the init function.
Also, read carefully what should `Waypoint` take in as parameters